ReplayKitExample/ReplayKitExample/PresentationViewController.swift

```diff
@@ -0,0 +1,715 @@
+//
+//  PresentationViewController.swift
+//  ReplayKitExample
+//
+//  Copyright © 2020 Twilio. All rights reserved.
+//
+
+import UIKit
+import TwilioVideo
+
+enum DataSourceError: Error {
+    // The room is not connected.
+    case notConnected
+}
+
+// MARK:- UIViewController
+class PresentationViewController : UIViewController {
+
+    static let kCellReuseId = "VideoCellReuseId"
+    static let kLargeCellSize = 160
+    static let kSmallCellSize = 78
+
+    var cameraSource: CameraSource?
+    var localVideoTrack: LocalVideoTrack?
+    var localAudioTrack: LocalAudioTrack?
+    var room: Room?
+
+    var remoteParticipants: [RemoteParticipant] = []
+
+    var statsTimer: Timer?
+    weak var remoteView: VideoView?
+    weak var scrollView: UIScrollView?
+    var accessToken: String?
+    weak var collectionView: UICollectionView?
+    weak var connectionLabel: UILabel?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+        collectionView.backgroundColor = nil
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        collectionView.contentInsetAdjustmentBehavior = .scrollableAxes
+        collectionView.register(VideoCollectionViewCell.self, forCellWithReuseIdentifier: PresentationViewController.kCellReuseId)
+        self.collectionView = collectionView
+
+        self.view.backgroundColor = .white
+
+        let label = UILabel()
+        label.text = "Connecting ..."
+        self.view.addSubview(label)
+        self.connectionLabel = label
+
+        self.view.addSubview(collectionView)
+        connectToPresentation()
+    }
```

Comment:
Should we add a hangup button for viewer?

Reply:
Good question. I didn't really want to clutter the UI so there is just the long press gesture.